quickstart/examples/sim/README.md

```diff
@@ -14,7 +14,7 @@ This is a simulation example that demonstrates how to deploy using the llm-d-inf
 ```bash
 # From the repo root
 cd quickstart
-HF_TOKEN=$(HFTOKEN) ./llmd-infra-installer.sh --namespace llm-d -r sim --gateway kgateway
+HF_TOKEN=$(HFTOKEN) ./llmd-infra-installer.sh --namespace llm-d-sim -r sim --gateway kgateway
 ```
 
     - It should be noted release name `sim` is important here, because it matches up with pre-built values files used in this example.
```

Comment:
```suggestion
    - It should be noted release name `infra-sim` is important here, because it matches up with pre-built values files used in this example.
```